index.html

```diff
@@ -1,16 +1,129 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <!-- Add a link to your css file here -->
-</head>
-
-<body>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
     <!-- Add your markup here -->
-</body>
+    <div class="content">
+      <header>
+        <div class="logo">
+          <img
+            class="logo-img"
+            src="img/logo.png"
+            alt="Cake image used as a logo"
+          />
+        </div>
+        <div>
+          <img src="img/menu-hamburger.svg" class="hamburger-menu" />
+        </div>
+        <p><i>The best cakes in town</i></p>
+        <p><i>delivered to your door</i></p>
```

Comment:
How could we have done this without putting the text in two separate containers for cleaner code?